branding/src/main/scala/com/normation/plugins/branding/BrandingConfService.scala

```diff
@@ -73,57 +77,66 @@ class BrandingConfService extends Loggable {
   val configFilePath = "/var/rudder/plugins/branding/configuration.json"
 
 
-  private[this] var cache : Box[BrandingConf] = reloadCache(true)
+  private[this] lazy val cache : Ref[Either[RudderError, BrandingConf]] = (for {
+    v <- Ref.make[Either[RudderError, BrandingConf]](Left(Unexpected("Cache is not yet initialized")))
+    c <- reloadCacheInternal(true, v).either
```

Comment:
I don't undertand how this can work. if file doesn't exist, we go to updateConf which NPE on cache.set, because cache isn't defined